### ISSUE TITLE
Group/Model tag color

### ISSUE THREAD

**Issue by cybercop23** [xLightsSequencer/xLights#4428]
Carry over the group and model tag color assigned in the layout to the import (left) grid, just as the sequencer tab has them.

**Comment by cybercop23** [comment]
Come to think of it, maybe carry this feature across all relevant areas, Model list and Edit Display Elements.

**Comment by cybercop23** [comment]
Hehehe.. you must have set the Tag color for that group. Just a nice and simpe way to organize groups and models. If you don't set the color, you'll never see it.

![image](https://github.com/xLightsSequencer/xLights/assets/23619433/37b91a1b-a8b0-4149-ac61-71ebed00987a)

**Comment by cybercop23** [comment]
Again, if you don't set the tag, you won't see anything. Yes, may be "some" processing time, but lets face it... with the wide screens that a lot of folks use, the little area doesn't take too much away and the main reason for it, is to help quickly find "things" as you scroll. The Master View is a huge mess/list and without (yet) a search/filter option things like this makes finding "special" items easier

**Comment by cybercop23** [comment]
Yup.. been there, done that... 
![image](https://github.com/xLightsSequencer/xLights/assets/23619433/f6b97e29-0534-446e-9423-683bfc373c80)

**Comment by cybercop23** [comment]
AGREE... kill it.. useless....
![image](https://github.com/xLightsSequencer/xLights/assets/23619433/a39e76c8-16bc-4bed-ac08-b0139357b3df)

**Comment by derwin12** [comment]
> The confusing part is I thought the colored squares were supposed to represent a Single Color string type. Now I think we have colored squares on the left for tags, on the right for single colored nodes, and then another color rectangle to show effects are on the group/model. Even if I don't use the tags they are going to be there when helping others. I need to double check what I saw because I don't think a tag color should have propagated down to the strand and node level. It was really distracting.

Colored squares on the right are the single color type you are talking about. On the left is the tag, not really used with smaller displays but coming in handy now with larger ones.

**Comment by derwin12** [comment]
Finding this isnt practical. The listctrl will put the > indicator at the start of the item and hence you cant neatly put the color tag (box or thick line) on the line. Anything I tried was cumbersome.

**Comment by cybercop23** [comment]
![Image](https://github.com/user-attachments/assets/26d10734-8ac5-494e-813b-12d8be7b0332)

**Comment by derwin12** [comment]
Careful..you have to fix all the loading   and saving maps and the sup
mapping...

**Comment by cybercop23** [comment]
Good point. Still need those sup files to test. Will try to get them later this morning if you're on zoom
